### ISSUE TITLE
mev-inspect not found error 

### ISSUE THREAD

**Issue by sbhadade** [flashbots/mev-inspect-py#310]
hi can somebody guide on this ?
![Screenshot 2022-08-24 at 20 41 59](https://user-images.githubusercontent.com/50355387/186455659-a07e7ab8-dedd-49ad-b3fe-a75ec4c70ef6.png)

**Comment by vyorkin** [comment]
+1 on this, I'd like to explore `mev-inspect`, but unfortunately, it is hard to follow the README at the moment

**Comment by mithradevik** [comment]
I have the same error coming up. Its really hard to follow up.
